.github/workflows/pages-deployment.yml

```diff
@@ -22,13 +22,8 @@ jobs:
           install-just: true
           python-version: '3.9'
 
-      - name: Upgrade pip
-        run: |
-          python3 -m pip install --upgrade pip
-
-      - name: Install dependencies
-        run: |
-          python3 -m pip install -r ./requirements.txt
+      - name: Install Python requirements
+        run: just devenv
```

Comment:
It looks like `just build` already does this?

Reply:
yes, you're right and I'm half-asleep! So we can just remove the lines, which is better.

Reply:
I'm greater than half asleep. Sorry for not spotting this. Thanks, George!